### ISSUE TITLE
Impact of salinity restoring

### ISSUE THREAD

**Issue by adele157** [adele157/IAF-meltwater#2]
Compute the effective runoff perturbation, after accounting for the surface salinity restoring. e.g. a time series similar to Figure 3c from [Moorman et al. 2020](https://journals.ametsoc.org/view/journals/clim/33/15/jcliD190846.xml). I guess integrate the salinity restoring change over the continental shelf region and add to the meltwater perturbation.

**Comment by matthew-england-unsw** [comment]
Yes and would be great to see the geographic distribution as well.   Could show longitude along x-axis, and then the MW anomaly and restoring flux kickback in y-axis, to check how it plays out relative to the MW source regions etc.

**Comment by adele157** [comment]
Why does this have such a weird pattern? Also can we convert it to a FW flux and plot the time series of meltwater input + salt restoring?

**Comment by adele157** [comment]
The restoring is reducing the meltwater perturbation by ~30%:
<img width="600" alt="Screen Shot 2022-09-15 at 11 37 45 am" src="https://user-images.githubusercontent.com/8506963/190293326-c160fe8a-471c-441d-89d1-afc1aadd5741.png">
